docs/teamshares/recipes.md

```diff
@@ -2,4 +2,5 @@
 
 - ## [Protips](/teamshares/recipes/protips.md)
 - ## [Rails UJS](/teamshares/recipes/rails-ujs.md)
+- ## [Converting Button View Component To Shoelace](/teamshares/recipes/buttons.md)
```

Comment:
@kdonovan updated naming here as well, do you think it's worth to change the file name? Or keep since this is the only recipe book for buttons

Reply:
Yeah, let's just tweak it to e.g. `shared-ui-buttons.md` :+1: